mkosi/__init__.py

```diff
@@ -568,6 +570,9 @@ def install_boot_loader(state: MkosiState) -> None:
         return
 
     if state.config.secure_boot:
+        assert state.config.secure_boot_key
+        assert state.config.secure_boot_certificate
```

Comment:
This should probably be checked earlier (and less often). Up until now this would blow up in `load_args`, so if we have to assert this now later, this seems like a small regression. I wonder if mypy could be placated to accept that keys exist if there were a dynamic validation check that checks that some dependent keys exist. Probably too difficult with the simple namespaces.

Reply:
We still check this in load_args()